packages/examples/nodejs/index.js

```diff
@@ -3,11 +3,13 @@ const fs = require('fs')
 
 const qrcode = require('qrcode-terminal');
 
-const options = {
-  shouldShimWeb3: false,
+const sdk = new MetaMaskSDK({
   storage: {
     enabled: true,
   },
+  dappMetadata: {
+    name: 'NodeJS example',
+  },
```

Comment:
isn't the `url` also required now?

Reply:
only one of the two is required. I removed mandatory url because it is not always used with RN